### DISCUSSION TITLE
Playbook

### DISCUSSION THREAD

**Discussion by joeb1kenobe** [Security-Onion-Solutions/securityonion#4867] [Unsupported Versions] [answered]
Created a new playbook today, but the elasticsearch config is not quite correct. Not sure if it is my playbook config or if the convert is not working right.

Playbook:
title: <title>
id: <guid>
status: experimental
description: <Description>
author: Joe Brown
date: 2021/07/20
tags:
    - attack.T1566
    - attack.T1566.002
    - attack.T1078
    - attack.T1078.002
    - attack.T1078.004
    - attack.T1056
    - attack.T1056.003
logsource:
    category: dns
detection:
    selection:
        query:
            - '45nklredpy0jqjtzchq.s3.us-east.cloud-object-storage.appdomain.cloud'
            - 'onlinedocfiles.neocities.org'
            - 'rebrand.ly'
    condition: selection
falsepositives:
    - Analyst testing 
level: high

Elastic generate:
query.security:("45nklredpy0jqjtzchq.s3.us\-east.cloud\-object\-storage.appdomain.cloud" OR "onlinedocfiles.neocities.org" OR "rebrand.ly")

would have expected it to be:
dns.query.name.security:("45nklredpy0jqjtzchq.s3.us\-east.cloud\-object\-storage.appdomain.cloud" OR "onlinedocfiles.neocities.org" OR "rebrand.ly")

Unless I am missing something.

Joe

**Comment by defensivedepth** [comment, marked as answer]
@joeb1kenobe It's because the field mappings are not set right. I have created an issue to get that fixed: https://github.com/Security-Onion-Solutions/securityonion/issues/4897

In the meantime, you can just specify the full field name in the selection clause - for example, instead of `query`, specify `dns.query.name`

**Reply by joeb1kenobe** [reply]
Awesome, Thanks.
